.github/workflows/ci.yml

```diff
@@ -1,5 +1,3 @@
-name: CI
-
```

Comment:
This looks like it might have accidentally gotten deleted in the merge.

Reply:
Thanks for the catch!